.github/workflows/bench.yml

```diff
@@ -16,13 +16,22 @@ jobs:
        target:
         - system: rpi4
           name: Arm Cortex-A72 (Raspberry Pi 4) benchmarks
-          cmd: tests bench -c PMU --arch-flags -mcpu=cortex-a72 -v --output output.json
+          bench_pmu: PMU
+          archflags: -mcpu=cortex-a72
+          cflags:
+          bench_extra_args:
         - system: rpi5
           name: Arm Cortex-A76 (Raspberry Pi 5) benchmarks
-          cmd: tests bench -c PERF --arch-flags "-mcpu=cortex-a76 -march=armv8.2-a" -v --output output.json
+          bench_pmu: PERF
+          archflags: "-mcpu=cortex-a76 -march=armv8.2-a"
+          cflags:
+          bench_extra_args:
```

Comment:
This can be removed if the default values are empty string for the two fields

Reply:
We are referring to those values below, though. I find it cleaner to set them to '' instead of relying on `${{ matrix.target.XXX }}` to default to the empty string if `XXX` is not set.